plugin/trino-hive/src/main/java/io/trino/plugin/hive/s3select/csv/S3SelectCsvRecordReader.java

```diff
@@ -104,4 +106,9 @@ public boolean shouldEnableScanRange()
                 buildInputSerialization().getCsv().getAllowQuotedRecordDelimiter());
         return CompressionType.NONE.equals(getCompressionType()) && !isQuotedRecordDelimiterAllowed;
     }
+
+    public static Optional<String> nullCharacterEncoding(Properties schema)
+    {
+        return Optional.ofNullable(schema.getProperty(SerdeConstants.SERIALIZATION_NULL_FORMAT, null));
```

Comment:
You can call the single argument version of `getProperty()` that returns a null by default